NPC/corner-turret/src/lib.rs

```diff
@@ -0,0 +1,78 @@
+extern crate wasmdome_mech_sdk as mech;
+
+use mech::*;
+
+mech_handler!(handler);
+
+pub fn handler(mech: impl MechInstruments) -> Vec<MechCommand> {
+    let mut cmds = Vec::new();
+
+    let corner = closest_corner(&mech);
+    if let Some(direction) = corner {
+        cmds.push(mech.move_mech(direction));
+    } else {
+        cmds.push(mech.fire_primary(fire_direction(&mech)));
```

Comment:
firing both primary and secondary in the same round should, I think, consume more than the 4 points allotted because firing the secondary is a 4-point action.